specification.md

```diff
@@ -2736,13 +2707,14 @@ schedule:
 
 </details>
 
-Any state can declare to be the start state of the workflow, meaning that when a workflow instance is created it will be the initial
+Any state can declare to be the start state of the workflow, meaning when a workflow instance is created it will be the initial
 state to be executed. A workflow definition can declare one workflow start state.
 
-The start definition provides a `kind` property, which describes the starting options:
-
-- **default** - The start state is always "active" and there are no restrictions imposed on its execution.
-- **scheduled** - Scheduled start states have two different choices. You can define time-based intervals during which workflow instances are **allowed**
+The start definition can be either `boolean` or `object` type.
```

Comment:
Wouldn't be better to add a new attribute to `start`, this way our tooling would know how to properly parse it? I mean instead of `try parse object else boolean`, we parse without conditionals. I think it will bring more complexity than it should.

Reply:
its about the language expression not the tooling complexity. you can try to parse it as boolean, and if its not then assume object.

"start": true and "end": true as default are much easier to understand for users imo

Reply:
Yes `start: end|true` is super easier to understand indeed. But just a note about parsing, some tooling will always parse "boolean" as false if it's not `true`. It's a nitpick, I agree we should prefer usability to implementation complexity.